packages/angular_devkit/build_angular/src/webpack/configs/typescript.ts

```diff
@@ -78,8 +77,13 @@ function createIvyPlugin(
   });
 }
 
-export function getNonAotConfig(wco: WebpackConfigOptions) {
-  const { tsConfigPath } = wco;
+export function getTypeScriptConfig(wco: WebpackConfigOptions) {
+  const { buildOptions, tsConfigPath } = wco;
+  const aot = !!buildOptions.aot;
+
+  if (aot) {
```

Comment:
We should always run this irrespective if it's AOT or JIT. Otherwise there is the risk that during JIT mode users use VE runtime instead of Ivy runtime, if the packages have not been processed by NGCC.